src/JuMP.jl

```diff
@@ -346,6 +346,10 @@ function setValue(v::Variable, val::Number)
     end
 end
 
+setValue(set::Array{Variable}, val::Array) = map(setValue, set, val)
+
+setValue(set::JuMPArray{Variable}, val::Array) = setValue(set.innerArray, val)
```

Comment:
This method only really makes sense for ``OneIndexedArray``. Otherwise you could do:
```
@defVar(m, x[[:a,:b,:c], 1:2])
setValue(x, [1,2,3,4,5,6])
```
which doesn't make too much sense.